pandas/tests/io/test_gcs.py

```diff
@@ -26,6 +26,21 @@ def test_read_csv_gcs(mock):
     assert_frame_equal(df1, df2)
 
 
+@td.skip_if_no('gcsfs')
+def test_to_csv_gcs(mock):
+    df1 = DataFrame({'int': [1, 3], 'float': [2.0, np.nan], 'str': ['t', 's'],
+                     'dt': date_range('2018-06-18', periods=2)})
+    with mock.patch('gcsfs.GCSFileSystem') as MockFileSystem:
+        s = StringIO()
+        instance = MockFileSystem.return_value
+        instance.open.return_value = s
+
+        df1.to_csv('gs://test/test.csv', index=True)
```

Comment:
Any particular reason you are explicitly stating `index=True` here instead of using the default?

Reply:
`df.to_csv(f)` and `pd.read_csv(f)` handle the index differently so I wanted to be extra clear that the index is also being checked in the round tripping